certified-connectors/DocumentTranslator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,907 @@
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Get translations status",
+        "description": "Gets a list of all translation requests submitted by a user and their status.",
+        "operationId": "GetTranslationsStatus",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important",
+            "description": "The value is the Azure key for your Translator or Cognitive Services resource.",
+            "x-ms-summary": "The value is the Azure key for your Translator or Cognitive Services resource."
+          }
+        ]
+      }
+    },
+    "/translator/text/batch/v1.0/batches/{id}/documents/{documentId}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string",
+                  "description": "path"
+                },
+                "sourcePath": {
+                  "type": "string",
+                  "description": "sourcePath"
+                },
+                "createdDateTimeUtc": {
+                  "type": "string",
+                  "description": "createdDateTimeUtc"
+                },
+                "lastActionDateTimeUtc": {
+                  "type": "string",
+                  "description": "lastActionDateTimeUtc"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "to": {
+                  "type": "string",
+                  "description": "to"
+                },
+                "progress": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "progress"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "characterCharged": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "characterCharged"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Get document status",
+        "description": "Gets the status for a specific document in a job.",
+        "operationId": "GetDocumentStatus",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The operation-ID",
+            "x-ms-summary": "The operation-ID",

```

Comment:
@HamshavathiM should this one be with a hyphen sign? Below I can see `document ID` without a hyphen.